include/grpcpp/impl/codegen/sync_stream.h

```diff
@@ -18,7 +18,7 @@
 #ifndef GRPCPP_IMPL_CODEGEN_SYNC_STREAM_H
 #define GRPCPP_IMPL_CODEGEN_SYNC_STREAM_H
 
-// IWYU pragma: private, include <grpcpp/support/sync_stream.h>
+// IWYU pragma: private, include <grpc/support/sync.h>
```

Comment:
This needs an exception, `sync_stream` is not a `sync` alternative.

Reply:
I think done before this review?

Reply:
Ah, you're right, I wasn't looking at the complete diff.